index.html

```diff
@@ -3842,6 +3846,107 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
 </aside>
 </section>
 
+<section class="informative changed"><h2>JSON Literals</h2>
+  <p>At times, it is useful to include JSON within JSON-LD that is not interpreted as JSON-LD.
+    Generally, a JSON-LD processor will ignore properties which don't map to <a>IRIs</a>,
+    but this causes them to be excluded when performing various algorithmic transformations.
+    But, when the data that is being described is, itself, JSON, it's important that
+    it survive algorithmic transformations.</p>
+
+  <p>By defining a term with <code>@type</code> <code>@json</code>,
+    a JSON-LD processor will treat the value as a <a>JSON literal</a>,
+    rather than interpreting it further as JSON-LD.
+    In the <a>expanded document form</a>, such JSON will become the value of <code>@value</code> within a <a>value object</a>
+    having <code>"@type": "@json"</code>.</p>
+
+  <p>When transformed into RDF, the JSON literal will have a lexical form based on
+    a specific serialization of the JSON,
+    as described in <a data-cite="JSON-LD11-API#round-tripping">Compaction algorithm</a> of [[JSON-LD11-API]].
+    and <a href="#the-rdf-json-datatype" class="sectionRef"></a>.</p>
```

Comment:
```suggestion
    and <a href="#the-rdf-json-datatype" class="sectionRef">the JSON datatype</a>.</p>
```
otherwise it reads "section X.Y.Z The JSON datatype", which is akward.